### PR TITLE
Fix various typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -L te`

### CODE REVIEW

File: install/mac/Stochas.pkgproj [hunk 1]
```diff
@@ -1608,14 +1608,14 @@
 		<dict>
 			<key>BACKGROUND</key>
 			<dict>
-				<key>APPAREANCES</key>
+				<key>APPEARANCES</key>
```
Comment (rudeog) [inline install/mac/Stochas.pkgproj:1611]: Most of these look good. However I am not sure we should change this pkgproj file as I doubt it was human edited, and I am not sure what the effect of changing this one file would be.

Comment (luzpaz) [reply install/mac/Stochas.pkgproj:1611]: Should i revert?

Comment (mthierman) [reply install/mac/Stochas.pkgproj:1611]: It's for the Mac installer right? It did pass the test. Not familiar with Mac installers though

Comment (baconpaul) [reply install/mac/Stochas.pkgproj:1611]: I had the same thought - just that one file leave unchanged. I want to change the max installer to the one which signs that we use elsewhere soon enough. But for now having this commit be comments only would be best

Comment (luzpaz) [reply install/mac/Stochas.pkgproj]: Done in [ac0d148](https://github.com/surge-synthesizer/stochas/pull/122/commits/ac0d148a8bc98cc505cd31872507b9f91c3fbe18)

Comment (baconpaul) [reply install/mac/Stochas.pkgproj]: Thanks! Will merge :)

**Pull request merged**